liota/lib/utilities/utility.py

```diff
@@ -110,6 +111,28 @@ def get_linux_version():
     return platform.platform()
 
 
+def get_default_network_interface():
+    """
+    Works with Linux.
+    :return: Default Network Interface of the Edge_System
+    """
+    cmd = "route | grep '^default' | grep -o '[^ ]*$'"
+    nw_iface = str(subprocess.check_output(cmd, shell=True)).rstrip()
+    log.info("Default network interface is : {0}".format(nw_iface))
+    return nw_iface
+
+
+def get_disk_name():
+    """
+    Works with Linux.
+    :return: Disk type of the Edge_System
+    """
+    cmd = "lsblk -io KNAME,TYPE | grep 'disk' | sed -n '1p' | grep -o '^\S*'"
```

Comment:
Thank you @warthog9 for pointing these out.  Tested these commands on DellGW 5000, RPi, Ubuntu, CentOS and RHEL VMs.  These utilities will be used when liota is collecting edge_system stats.  If an edge_system falls under these 'unusual' circumstance, I think it should be handled manually.